roles/mesos/tasks/follower.yml

```diff
@@ -78,6 +78,15 @@
   tags:
     - mesos
 
+- name: set follower resources
+  sudo: yes
+  copy:
+    dest: /etc/mesos-slave/resources
+    content: "{{ mesos_resources }}"
+  notify: restart mesos follower
```

Comment:
Does that mean change:

```
- name: remove mesos follower metadata
  sudo: yes
  shell: "{{ item }}"
  with_items:
    - /bin/systemctl stop mesos-slave
    - /bin/rm -rf /tmp/mesos/meta
    - /bin/systemctl start mesos-slave
```

to

```
- name: remove mesos follower metadata
  sudo: yes
  shell: "{{ item }}"
  with_items:
    - /bin/systemctl stop mesos-slave
    - /bin/rm -rf /tmp/mesos/meta/slaves/latest
    - /bin/systemctl start mesos-slave
```

Or were you thinking a new handler that just removes `/tmp/mesos/meta/slaves/latest` ?